packages/app/src/cli/models/extensions/specifications/editor_extension_collection.ts

```diff
@@ -2,16 +2,17 @@ import {BaseSchema} from '../schemas.js'
 import {createExtensionSpecification} from '../specification.js'
 import {zod} from '@shopify/cli-kit/node/schema'
 
-export interface InCollection {
+interface IncludeSchema {
   handle: string
 }
 
-const InCollectionSchema = zod.object({
+const IncludeSchema = zod.object({
   handle: zod.string(),
 })
 
 export const EditorExtensionCollectionSchema = BaseSchema.extend({
-  in_collection: zod.union([zod.array(InCollectionSchema), zod.array(zod.string())]).optional(),
+  include: zod.array(IncludeSchema).optional(),
```

Comment:
Both optional so you can use either one of them. We will add validation to make sure at least one of them exists